porcupine/_state.py

```diff
@@ -7,9 +7,15 @@
 import tkinter
 import types
 from typing import Any, Type
+from sys import platform
```

Comment:
There's a mypy bug that prevents this from working :D https://github.com/python/mypy/issues/10054

You need to use `import sys` and `sys.platform`.